spec/features/proposal/collection_draft_proposals/collection_draft_proposal_reject_spec.rb

```diff
@@ -0,0 +1,145 @@
+          expect(page).to have_content('Please provide the reasons for rejecting this proposal submission.')
+          # field for reasons
+          # field for note
+          # expect(page).to have_select('Reason(s) for Rejection', with_options: ['Missing Keywords', 'Insufficient Content', 'Misspellings/Grammar', 'Invalid/Incorrect Content', 'Broken Links', 'Duplicate Metadata', 'Other'])
+          expect(page).to have_select('proposal-rejection-reasons', with_options: ['Missing Keywords', 'Insufficient Content', 'Misspellings/Grammar', 'Invalid/Incorrect Content', 'Broken Links', 'Duplicate Metadata', 'Other'])
+          expect(page).to have_field('Note')
+        end
+      end
+
+      context 'when attempting to reject without entering feedback' do
+        before do
+          within '#reject-submission-modal' do
+            click_on 'Reject'
+          end
+        end
+
+        it 'displays validation errors' do
+          within '#reject-submission-modal' do
+            expect(page).to have_css('.eui-banner--danger', count: 2)
+            expect(page).to have_content('Reason(s) are required')
+            expect(page).to have_content('Note is required')
+          end
+        end
+
+        context 'when then entering feedback' do
+          before do
+            within '#reject-submission-modal' do
+              select 'Broken Links', from: 'proposal-rejection-reasons'
+              fill_in 'Note', with: 'There are many reasons for rejecting this submission'
+
+              all('label[for="rejection_note"]').first.click
+            end
+          end
+
+          it 'does not display the validation errors' do
+            within '#reject-submission-modal' do
+              expect(page).to have_no_css('.eui-banner--danger')
+            end
+          end
+        end
+      end
+
+      context 'when rejecting the proposal with feedback' do
+        before do
+          within '#reject-submission-modal' do
+            select 'Broken Links', from: 'proposal-rejection-reasons'
+            fill_in 'Note', with: 'There are many reasons for rejecting this submission'
+
+            click_on 'Reject'
+          end
+
+          it 'rejects the proposal' do
+            expect(page).to have_content('Collection Draft Proposal Rejected Successfully')
+            within '#proposal-status-display' do
+              expect(page).to have_content('Draft Proposal Submission')
+              expect(page).to have_content('Rejected')
+            end
+          end
+        end
+      end
+
+      context 'when clicking Cancel' do
+        before do
+          click_on 'Cancel'
+        end
+
+        it 'does not navigate away when cancelling' do
+          within '#proposal-status-display' do
+            expect(page).to have_content('Draft Proposal Submission')
+            expect(page).to have_content('Submitted')
+          end
+        end
+      end
+    end
+
+    context 'when the approving fails' do
```

Comment:
The context and comment should probably be changed to make sense for this file.

Reply:
addressed